tools/reactive-controllers/src/RovingTabindex.ts

```diff
@@ -74,6 +74,26 @@ export class RovingTabindexController<
         });
     }
 
+    changeDefaultItemFocus(): void {
+        const currentIndex = this.currentIndex;
+        let focusIndex = currentIndex;
+        if (currentIndex < this.elements.length - 1) {
+            for (let i = currentIndex + 1; i < this.elements.length; i++) {
+                focusIndex = i;
+                break;
+            }
+        } else if (currentIndex > 0) {
+            for (let i = currentIndex - 1; i >= 0; i--) {
+                focusIndex = i;
+                break;
+            }
+        }
```

Comment:
Without logic inside of the `for` statements, won't this always get simply the next or previous item, regardless of whether those items are focusable or not?

Also, if `(currentIndex < this.elements.length - 1)` and all subsequent items are NOT focusable, what element should become focusable?

Reply:
1. No the next or previous item doesn't get the focus without the logic inside `for` statements
2. If the current index is not the last element and if all the next items are not focusable then the focus should remain on the last element if its not disabled?

Reply:
What I thinking, and maybe this is just a side effect of me only reviewing this first thing in the morning, is that if `currentIndex === 5` and `this.elements.length === 10`, the logic above will also set `focusIndex` as `6` whether it's focusable or not?

Similarly, `currentIndex === 5` and `this.elements.length === 6`, this will also set `focusIndex` as `4`... 

Maybe resolving past that is managed elsewhere?